main.py

```diff
@@ -112,6 +112,15 @@ def update(self) -> Image:
             'sha256:9a4516444fef9e0f11ee6b2de716d3b97b36bf05d9cc2d44c4596cfb0584dea6',  # noqa: E501
         ),
     ),
+    Image(
+        registry='registry-1.docker.io',
+        source='beezz/redis-cluster',
```

Comment:
this should be a sentry repo if we're not going to use the upstream.  we can push directly to ghcr instead of using the image mirror for that